### ISSUE TITLE
Metric for the total parameters

### ISSUE THREAD

**Issue by Charleshhy** [KMnP/vpt#10]
Thanks for the great work! 

I noticed that you are using Total Params as one metric in your paper to measure the trainable parameters, as below.

<img width="529" alt="image" src="https://user-images.githubusercontent.com/39027479/190891247-1ab42b2e-ac56-46cb-805d-ded9d5194671.png">

However, I am quite confused about how to derive these scores. For example, LINEAR only tunes the classification heads, so the total trainable parameters should be (sum_of_classes * 768 + sum_of_classes = 0.72M) for the 19 VTAB dataset added together, while FULL should have (85.8 * 19 = 1630.2M) trainable parameters. It seems a little bit far from 1.01x for LINEAR and 19.01x for FULL finetuning.

Besides, is it possible to share the number of prompts for each task used to get the results in Table 4?

Kind regards,
Charles

**Comment by KMnP** [comment]
Hi Charles,

Thank you for your interest in our work! 

"Total params" are defined as total parameters needed for all the downstream tasks evaluated in the table. For LINEAR, total params will be the total trainable params you mentioned + total parameters of one ViT-B backbone = 1.01 * one ViT-B backbone params.

As for Table 4, you could find the hyper-parameters [here](https://github.com/KMnP/vpt#hyperparameters-for-experiments-in-paper)

**Comment by Charleshhy** [comment]
Thanks a lot, that solves all of my questions.

**Comment by andytu28** [comment]
@KMnP Thank you so much for the great work and the reply on this issue. 

I am trying to find the hyper-parameters for Table 4 (results for MAE and MoCoV3 backbones). However, I could not find them in the vtab_all_release.csv file you provided in the GoogleDrive and Dropbox links. I can only see the hyper-parameters for supervised backbones, such as sup_vitb16_imagenet21k, swinb_imagenet22k_224, ... etc. Did I miss something, or could you please help share the hyper-parameters for the self-supervised backbones, like MAE and MoCoV3?
